Source/WebKit/UIProcess/PageLoadState.h

```diff
@@ -152,6 +159,8 @@ class PageLoadState {
     bool hasNegotiatedLegacyTLS() const;
     void negotiatedLegacyTLS(const Transaction::Token&);
     bool wasPrivateRelayed() const { return m_committedState.wasPrivateRelayed; }
+    String proxyName() { return m_committedState.proxyName; }
+    WebCore::ResourceResponseBaseSource source() { return m_committedState.source; }
```

Comment:
We should probably rename this enum to remove "Base" from its name.  That is a confusing implementation detail of how we make ResourceResponse implemented on different platforms.

Reply:
Good idea. Done.